documentation/pages/move-registry/tooling/mvr-cli.mdx

```diff
@@ -0,0 +1,129 @@
+import { Tabs } from 'nextra/components'
+import { Callout } from 'nextra/components'
+import Beta from '../../../components/beta'
+
+<Beta/>
+
+# MVR CLI
+
+Use the MVR CLI to manage your package dependencies and to interact with the Move Registry (MVR). The MVR CLI is a command line tool that enables:
+
+- Adding dependencies to your Move project.
+- Building your Move project with MVR dependencies.
+
+## Installation
+
+There are several ways to install the MVR CLI tool.
+
+### Cargo
+
+Use the following command to install using Cargo:
+
+```sh
+cargo install --locked --git https://github.com/mystenlabs/mvr --branch release mvr
+```
+
+### From binaries
+
+Download the correct binary file for your OS and architecture from the [MVR GitHub release page](https://github.com/MystenLabs/mvr/releases/latest):
+
+After downloading the file, rename the binary to `mvr` and add its location to your system `PATH` variable.
+
+
+<Tabs items={['Zsh', 'Bash', 'fish', 'Windows']}>
```

Comment:
I think we could leave this out. There is plenty of documentation on adding binaries to your PATH on various architectures.

Reply:
Leaving for now. It's more of a tool to jog memory than detailed instruction. It's also easy enough for reader to skip over if not needed. Having said that, ask me again and I'll remove because I'm not married to it.